assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -64,17 +96,25 @@ export default function WidgetAreaRenderer( { slug, totalAreas } ) {
 		select( CORE_WIDGETS ).isWidgetAreaActive( slug )
 	);
 
+	const viewContext = useSelect( ( select ) =>
+		select( CORE_UI ).getValue( ACTIVE_CONTEXT_ID )
```

Comment:
@eugene-manuilov please be sure to update the name where it's selected too 👍 

```suggestion
	const activeContextID = useSelect( ( select ) =>
		select( CORE_UI ).getValue( ACTIVE_CONTEXT_ID )
```